src/Home.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from "react";
-import Slider from './Slider/Slider.js'
-import Movies from './Allmovies/allmovies.js'
+import Slider from './Slider/Slider'
+import SearchForm from './SearchForm/SearchForm';
+import Movies from './Allmovies/AllMovies'
+import FilmStore from './Allmovies/Film-store'
```

Comment:
for json files will be better to use dash-case. please see comment from previous PR.